bee-common/bee-common/src/logger/config.rs

```diff
@@ -58,6 +70,8 @@ pub struct LoggerOutputConfig {
     pub(crate) name: String,
     /// Log level of an output.
     pub(crate) level: LevelFilter,
+    /// Log filters of the output.
+    pub(crate) filters: Option<Vec<String>>,
```

Comment:
IMHO with another way of filtering log output we should refactor the naming a bit. I'ld rename `level` to `level_filter` and `filters` to `target_filters`. I think I'ld also introduce a typedef `type TargetFilters = Vec<String>`, but you can ignore this if you don't like to do this. I'ld also rename all the fields accordingly. They are not that short anymore, but I think it helps with clarity.